File: powtorzenie/wej_wyj.cpp
/*
 * wej_wyj.cpp
 *
 */

#include <iostream>

using namespace std;

int main(int argc, char **argv)
{
    char imie[10], nazwisko[20];
        
    cout << "Jak masz na imię?     ";
    cin >> imie;
        
    cout << "Jak masz na nazwisko?     ";
    cin >> nazwisko;
    
    cout << "Witaj" << " " << imie << " " << nazwisko << "!" << endl;

	return 0;
}

File: powtorzenie/wej_wyj.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
#
#  wej_wyj.py
#  

def main(args):
    
    imie = input("Jak masz na imię?     ")
    nazwisko = input("Jak masz na nazwisko?     ")
    
    print("Witaj", imie, nazwisko, "!")
    
    return 0

if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
